dnx/src/Microsoft.DotNet.xunit.performance.runner.dnx/project.json

```diff
@@ -12,7 +12,7 @@
   "requireLicenseAcceptance": false,
 
   "dependencies": {
-    "Microsoft.DotNet.xunit.performance.run.core": "99.99.99-dev",
+    "Microsoft.DotNet.xunit.performance.run.core": "1.0.0-alpha-build0018",
```

Comment:
Ah, I see. I was locally deploying the packages and they weren't being updated so I manually went in and modified them. That explains a bit though, thanks.